Samples/csharp_dotnetcore/01.federated-search-provider/Constants.cs

```diff
@@ -25,6 +25,29 @@ public class Constants
         /// </summary>
         public const string Authority = "https://login.microsoftonline.com/";
 
+        /// <summary>
+        /// Resource ID of target API which will be used to generate token which target API will allow to access
```

Comment:
This feels less grammatical than I would want for public documentation. 

Maybe something like
Resource ID. This resource id will be included in the generated token. The target API should be configured to allow access to requests with this resource id.

Reply:
This resource ID is resource ID of targeted API. So when 3S want to call Cortex with OBO it will get token with https://cortana.ai resource id. I will try to reword it.